crates/egui/src/containers/frame.rs

```diff
@@ -29,6 +29,8 @@ pub struct Frame {
 
     pub fill: Color32,
 
+    pub selected_header_color: Color32,
```

Comment:
Unused?

Reply:
Yes unused. Sorry, it was part of my early experiments. I removed it.